instrumentation-genai/opentelemetry-instrumentation-langchain/tests/test_span_manager.py

```diff
@@ -0,0 +1,84 @@
+import unittest.mock
+import uuid
+
+import pytest
+
+from opentelemetry.instrumentation.langchain.span_manager import (
+    SpanManager,
+    _SpanState,
+)
+from opentelemetry.trace import SpanKind, get_tracer
+from opentelemetry.trace.span import Span
+
+
+class TestSpanManager:
```

Comment:
Please just add a test case for the memory cleanup you added with `del`, if you don't have.